crates/bevy_window/src/lib.rs

```diff
@@ -22,14 +22,14 @@ use bevy_app::{prelude::*, Events};
 
 pub struct WindowPlugin {
     pub add_primary_window: bool,
-    pub exit_on_close: bool,
+    pub exit_method: WindowExitMethod,
 }
```

Comment:
Is `bevy_window` meant to work apart from `bevy_winit`? If so, I think this should be reverted. Since closing by `WindowExitMethod` is controlled by `bevy_winit`, it might make sense to leave this option in for the pure `bevy_window` (and instead maybe just default the value to `false`).